recipes/tetools_l-modolo/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin
+sed -i 's|#!/usr/bin/Rscript|#!/usr/bin/env Rscript|' TEdiff.R
+sed -i 's|#!/usr/bin/python3|#!/usr/bin/env python|' TEcount.py
+sed -i 's|#!/usr/bin/python3|#!/usr/bin/env python|' PingPong.py
```

Comment:
Try replacing with the result of `which python` and `which Rscript`. That would allow the installed scripts to run as e.g. `~/miniconda3/envs/tetools/TEcount.py` without activating the environment. Conda will replace the path to the interpreter upon install. Alternatively, a minimal `setup.py` should do the right thing, too.